config/application.rb

```diff
@@ -37,5 +37,13 @@ class Application < Rails::Application
     config.importmap.cache_sweepers << Rails.root.join('app/components')
 
     config.view_component.default_preview_layout = 'lookbook'
+
+    config.active_record.schema_format = :sql
+
+    # Version has_many and belongs_to associations (This feature is experimental due to the number of edge cases)
```

Comment:
**nitpick (blocking):** Can we uncomment the following logidize options which means that you can delete `config/intializers/logidze.rb`. Or delete the comments and add the `ignore_log_data_by_default` option to `config/initializers/logidze.rb`

This will also require you to use `reload_log_data` in your tests that access `log_data`. https://github.com/palkan/logidze#dealing-with-large-logs